### PR TITLE
BMD-6702 - Input moved out of folded lists

### PR DESCRIPTION
**Pull request by bmxjusu**
### Jira ticket:
https://berlingskemedia.atlassian.net/browse/BMD-6702
### Pre/Post release actions:
(none)

### CODE REVIEW

File: client/src/components/ArrayItems.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export function ArrayItems(props: ArrayItemsProps): JSX.Element {
   });
 
   React.useEffect(() => {
+    console.log(props.data);
```
Comment (bmxsele) [inline client/src/components/ArrayItems.tsx:46]: You forgot about this console.log probably :smile: It shouldn't stay in the code, I'd suggest removing it

**Review by bmxsele — CHANGES_REQUESTED** [review]

File: client/src/components/ArrayItems.tsx [hunk 2]
```diff
@@ -203,16 +207,52 @@ export function ArrayItems(props: ArrayItemsProps): JSX.Element {
...
             <tbody>{renderItems()}</tbody>
           </table>
-          <div
-            style={{
-              backgroundColor: '#f9f9f9',
-              padding: '15px 10px 0',
-              marginBottom: '20px',
-            }}>
-            {state.showTruncatedList ? (
+          {state.showTruncatedList ? (
```
Comment (bmx-makostecka) [inline client/src/components/ArrayItems.tsx:249]: Now you duplicate a lot of code. I suggest changing it a little bit, something like that (I simplified it just for showing you what I mean 🙂 ):
`{props.data.length > 5 && (
    <div>
        <button onClick={toggleFullList}>
            {state.showTruncatedList ? 'Show' : 'Hide'} {props.data.length} items
        </button>
    </div>
)`
and of course, you need to change `hideFullList` and `showFullList` to `toggleFullList`

Comment (bmx-makostecka) [reply client/src/components/ArrayItems.tsx:249]: Justus, if you have any questions about what I wrote - ask me on slack 🙂

Comment (bmxjusu) [reply client/src/components/ArrayItems.tsx:249]: Good point! I changed it now :) Thank you for the feedback

**Review by bmx-makostecka — DISMISSED** [review]

**Review by bmxsele — APPROVED** [review]

File: client/src/components/ArrayItems.tsx [hunk 3]
```diff
@@ -203,16 +206,52 @@ export function ArrayItems(props: ArrayItemsProps): JSX.Element {
...
             <tbody>{renderItems()}</tbody>
           </table>
-          <div
-            style={{
-              backgroundColor: '#f9f9f9',
-              padding: '15px 10px 0',
-              marginBottom: '20px',
-            }}>
-            {state.showTruncatedList ? (
+          {props.data.length > 5 && (
```
Comment (bmxsele) [inline client/src/components/ArrayItems.tsx:248]: We can optionally create also a constant for this value `5` and call it for example: `TRUNCATED_LIST_SIZE` or something like that, but i think it's not required so i gave approval

**Review by bmxsele — APPROVED** [review]

**Pull request merged**